cnd/src/facade.rs

```diff
@@ -137,14 +146,46 @@ impl<B: ledger::bitcoin::Bitcoin + 'static> HtlcEvents<B, asset::Bitcoin> for Fa
 }
 
 #[async_trait::async_trait]
-impl<A> HtlcEvents<Ethereum, A> for Facade
-where
-    A: Asset + Send + Sync + 'static,
-    ethereum::Cache<Web3Connector>: HtlcEvents<Ethereum, A>,
-{
+impl HtlcEvents<Ethereum, asset::Ether> for Facade {
```

Comment:
1. Because the other thing is no longer allowed by the compiler now that the type we are abstracting over (`Asset`) lives in a different crate.
2. In the longer run, I'd hope to use the ambassador crate to create this impls and those will always be concrete forwardings.
3. I think it is actually more correct to be explicit here instead of trying to be clever and use generics. I'd rather use something like ambassador to avoid us having to type this code instead of using generics.